### DISCUSSION TITLE
Having contribution guide?

### DISCUSSION THREAD

**Discussion by hayata-yamamoto** [okaryo/TabTabTab#382] [Q&A] [answered]
I have some questions on contributions 

- Does this repository open to outsider contribution?
- Is there contribution guide? ex.) CONTRIBUTING.md or something 
- Is there setup guide? ex.) `npm i` or something

**Comment by okaryo** [comment, marked as answer]
Hi,

Yes, I do accept external contributions. However, for adding new features or improvements, it would be great if you could discuss them first in issues or discussions before submitting a pull request. I’ll work on preparing a contribution guide and setup guide as soon as possible, though due to my work commitments, I haven’t had much time lately. I aim to have them ready by the end of the year, so please stay tuned. Thanks for your patience! 🙏

**Reply by hayata-yamamoto** [reply]
Thanks, I look for
